logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -619,6 +620,7 @@ internal class MLSConversationDataSource(
         externalSenders: ByteArray,
         allowPartialMemberList: Boolean = false,
     ): Either<CoreFailure, MLSAdditionResult> = withContext(serialDispatcher) {
+        kaliumLogger.d("establish MLS group: $groupID")
```

Comment:
is it fine to log the full group id here ?

Reply:
yes. there is no harm in it, and it's a debug log.